libs/dh/market-participant/actors/feature-actors/src/lib/drawer/dh-actor-drawer.component.ts

```diff
@@ -102,4 +110,25 @@ export class DhActorDrawerComponent {
 
     return emDash;
   }
+
+  get gridAreaOrFallback() {
+    return this.actor?.gridAreas?.[0]?.code ?? emDash;
+  }
+
+  private loadActor(id: string): void {
+    this.subscription?.unsubscribe();
+
+    this.getActorByIdQuery$.setVariables({ id });
+
+    this.subscription = this.getActorByIdQuery$.valueChanges
+      .pipe(takeUntil(this.closed))
+      .subscribe({
+        next: (result) => {
+          this.actor = result.data?.actor;
+        },
+        error: (error) => {
+          console.log('error', error);
```

Comment:
Suggestion: Maybe show the an error in the UI?

Reply:
I was actually considering removing the error handler, because most of the data is already available from the table (using partial loading) and the rest has an em dash fallback in case it fails to load.

Do you think we should still show some kind of an error message?

Reply:
Sounds like it is handled